README.md

```diff
@@ -54,6 +54,63 @@ Example configuration:
     ```
     See the [Kafka Connect transforms](https://kafka.apache.org/31/documentation.html#connect_transforms) documentation for more details.
 
+**Note:** For each message record that the sink connector writes to Cloudant the document ID from this priority order:
+
+1. From the value of the `cloudant_doc_id` header on the message.  The value passed to this header must be a string and the `header.converter=org.apache.kafka.connect.storage.StringConverter` config is required.  This will overwrite the `_id` field if it already exists.
+1. The value of the `_id` field in the JSON
+1. If no other non-null or non-empty value is available the document will be created with a new UUID.
+
+#### Single Message Transforms
+
+Single Message Transforms, or SMTs, can be used to customize fields or values of messages during data flow.  The examples below will explore modifying fields for messages flowing from the Kafka topic to a Cloudant database using the sink connector.
+
+1. If there's an existing field that will be used as the document ID then use the `ReplaceField` transform.  See the config example below that renames `name` to `_id` field:
```

Comment:
Add a comma to make these clauses clearer: "If there's an existing field. that..." or even "If there's an existing field, _then_ that..."

Reply:
Ok, I'll use what Rich has above.  I also realized that the transform in this note should be `RenameField`.